contrib/meson.build

```diff
@@ -12,6 +12,19 @@ configure_file(
   configuration: con2,
 )
 
+uswid = find_program('uswid', required: false)
+if uswid.found()
+  custom_target('gen-sbom',
+    input: 'sbom.cdx.json',
+    output: 'sbom.cdx.json',
```

Comment:
should this install into the filesystem somewhere and not just build?

Reply:
I'm pondering that -- I was going to use it in a container build like we do the metainfo.xml file but for the UEFI usecase we just need to discover it from the git checkout. e.g. https://blogs.gnome.org/hughsie/2024/11/14/firmware-sboms-for-open-source-projects/